python/src/QuantileConfidence_doc.i.in

```diff
@@ -0,0 +1,423 @@
+tail : bool, optional
+    Whether to return the lower or upper tail confidence interval. Default is False.
+
+Returns
+-------
+ci : :class:`~openturns.Interval`
+    Quantile confidence interval
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> alpha = 0.05
+>>> beta = 0.95
+>>> algo = otexp.QuantileConfidence(alpha, beta)
+>>> sample = ot.Gumbel().getSample(100)
+>>> ci = algo.computeUnilateralConfidenceInterval(sample)
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileConfidence::computeBilateralConfidenceInterval
+"Evaluate a bilateral confidence interval of a quantile.
+
+The confidence interval is given by the order statistics:
+
+.. math::
+
+    [X_{(k_1)}, X_{(k_2)}]
+
+where the ranks :math:`k_1, k_2` are given by :meth:`computeBilateralRank`
+
+Parameters
+----------
+sample : 2-d sequence of float
+    Sample of the variable :math:`X`
+
+Returns
+-------
+ci : :class:`~openturns.Interval`
+    Quantile confidence interval
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> alpha = 0.05
+>>> beta = 0.95
+>>> algo = otexp.QuantileConfidence(alpha, beta)
+>>> sample = ot.Gumbel().getSample(60)
+>>> ci = algo.computeBilateralConfidenceInterval(sample)
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileConfidence::computeUnilateralMinimumSampleSize
+"Evaluate the minimum size of the sample for the unilateral case.
+
+When the rank :math:`r` is zero the minimum sample size is immediate for the lower bound:
+
+.. math::
+
+    n_{low} = \frac{\log(1-\beta)}{\log(1-\alpha)}
+
+and for the upper bound:
+
+.. math::
+
+    n_{up} = \frac{\log(1-\beta)}{\log(\alpha)}
+
+Else in the general case we search for the smallest :math:`n` which verifies:
+
+.. math::
+
+    F_{n,\alpha}(n-r-1) \geq \beta
+
+with :math:`F_{n,\alpha}` the cumulative distribution function of the Binomial distribution.
+
+Parameters
+----------
+rank : int, optional
+    Rank of the quantile. Default value is 0.
+tail : bool, optional
+    Whether to return the lower or upper tail minimum size. Default is False.
+
+Returns
+-------
+size : int
+    Minimum sample size
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> alpha = 0.05
+>>> beta = 0.95
+>>> algo = otexp.QuantileConfidence(alpha, beta)
+>>> size = algo.computeUnilateralMinimumSampleSize(0)
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileConfidence::computeBilateralMinimumSampleSize
+"Evaluate the minimum size of the sample for the bilateral case.
+
+The minimal bilateral sample size is the smallest integer :math:`n` such that:
+
+.. math::
+
+    1-\alpha^n-(1-\alpha)^n \geq \beta
+
+if and only if
+
+.. math::
+
+    \Pset(X_(1) \leq x_{\alpha} \leq X_({n})) \geq \beta
```

Comment:
This is the rendering:

![image](https://github.com/user-attachments/assets/2253d3ee-b008-4d0c-98af-247e1ec9f030)

```suggestion
    \Pset(X_{(1)} \leq x_{\alpha} \leq X_{(n)}) \geq \beta
```